circup/__init__.py

```diff
@@ -46,6 +49,9 @@
 LOG_DIR = appdirs.user_log_dir(appname="circup", appauthor="adafruit")
 #: The location of the log file for the utility.
 LOGFILE = os.path.join(LOG_DIR, "circup.log")
+#: The localtion to store a local copy of code.py for use with --auto and
```

Comment:
```suggestion
#: The location to store a local copy of code.py for use with --auto and
```

Reply:
this and a few other changes are in the latest commit, I'm marking them resolved here so I can refer back to the page and work on the others.